### DISCUSSION TITLE
V4L2 NV Extensions API

### DISCUSSION THREAD

**Discussion by chhg82** [OE4T/meta-tegra#1407] [Q&A] [answered]
Hi,
I want to apply the [NVIDIA API](https://docs.nvidia.com/jetson/l4t-multimedia/group__ee__extensions__group.html) on the `kirkstone-l4t-r32.7.x` branch. In meta-tegra I only find `v4l2_nv_extensions.h` in the `gstreamer1.0-plugins-nvvideo4linux2-src`.
What did I miss?
Thanks!

**Comment by madisongh** [comment, marked as answer]
That header is in the Jetson Multimedia API kit, but our `tegra-mmapi` and `tegra-mmapi-samples` recipes don't install it anywhere for use by other recipes.
